tests/test_home_page.py

```diff
@@ -1,23 +1,33 @@
+'''
+Checks functionality of HomePage
+'''
 from utilities.base_test_case import BaseTestCase
 from pages.home_page import HomePage
-import time
 
 
 class TestOpenHomePage(BaseTestCase):
+    '''
+    Class for HomePage tests
+    '''
 
-    def test_OpenHomePage(self):
+    def test_open_home_page(self):
         '''
         Verifies if the HomePage opens
         '''
-        assert self.driver.title == "Google"
+        driver = self.driver
+        home_page_title = "Google"
+        self.assertEqual(driver.title, home_page_title, "HomePage Failed to load")
 
-    def test_GoogleSearch(self):
+    def test_google_search(self):
         '''
         Verifies if page with search results opens
         '''
-        home_page = HomePage(self.driver)
+        driver = self.driver
+        home_page = HomePage(driver)
+
         home_page.get_search_field().send_keys('apple')
```

Comment:
What if the search field is not yet loaded? In majority of cases explisit waits help to handle this. 
That's why I suggested you to create a parent class for page object classes - to store there methods for all pages, for example:

- __init__()
- wait_for_element_to_load(selector: tuple, timeout=10)

This way you can use it in other methods of page objects or directly in tests